doc/CCPPtechnical/source/ConstructingSuite.rst

```diff
@@ -160,16 +160,16 @@ Consider the case where a model requires that some subset of physics be called o
      </group>
    </suite>
 
-------------------------------- 
+-------------------------------
 Operational GFS v15 Suite
```

Comment:
I am still not sure why the GFS suite is documented in the CCPP Framework which is supposed to be infrastructure for all models. Are all suites going to be documented here?

Reply:
We briefly touched on this issue of having documentation that changes more frequently under ccpp-framework. The problem I think is that the "big document" aka techdoc combines technical aspects of the ccpp-framework and scientific aspects such as the suites.

We agreed that it would be better to move this contents into a ccpp-doc (ccpp-documentation) repository on the NCAR GitHub to avoid changing ccpp-framework everytime a small change is made to the documentation. Since the techdoc is NOAA-NCAR/DTC centric, this would be something NCAR/CGD NCAR/MMM don't really have to worry about. We also agreed that your feedback is valuable and that it still would be good to invite you for looking at the PRs. What do you think? Details could be fleshed out next Thursday?

Reply:
I think I agree on all these fronts. The CCPP Framework documentation should describe some simple suite or suites that demonstrate the features of suites.
The ccpp-doc would be a more complete technical document for GMTB / NOAA.
Ideally, we could configure the CCPP Framework technical document to make it easy to incorporate into model-specific docs (e.g., UFS, CESM). I would also like to see some way to have a technical document for a suite that is part of the suite when we finally stand up a CCPP Physics repository.
I am happy to review other documents (when I'm not too busy :).